images/ubuntu/scripts/build/install-azure-devops-cli.sh

```diff
@@ -9,7 +9,30 @@
 export AZURE_EXTENSION_DIR=/opt/az/azcliextensions
 echo "AZURE_EXTENSION_DIR=$AZURE_EXTENSION_DIR" | tee -a /etc/environment
```

Comment:
These lines aren't needed, but were introduced by the refactoring of the build scripts.

The environment variable is already set in `install-azure-cli.ch` and it makes more sense there, as it's a config setting of `az-cli` and not of `az-devops`.